### ISSUE TITLE
benchmarking for TP llms

### ISSUE THREAD

**Issue by AleksKnezevic** [tenstorrent/tt-xla#2603]
Please add support for llama 3.2 8b and qwen 3 32B to tt-forge benchmarks.

**Comment by AleksKnezevic** [comment]
@btsoiTT, can you please take over the remainder of this task. @gengelageTT already added llama 3 8b [here](https://github.com/tenstorrent/tt-forge/pull/760), you can continue with qwen 3 32b, plus any remaining TP red LLMs. @kmabeeTT can give you a list.

**Comment by btsoiTT** [comment]
https://github.com/tenstorrent/tt-forge/pull/805 adds the following 13 models

```
falcon/pytorch-tiiuae/Falcon3-7B-Base
falcon/pytorch-tiiuae/Falcon3-10B-Base
llama/causal_lm/pytorch-llama_3_1_8b_instruct
mistral/pytorch-7b_instruct_v03
mistral/pytorch-ministral_8b_instruct
mistral/pytorch-mistral_nemo_instruct_2407
qwen_2_5/causal_lm/pytorch-14b_instruct
qwen_3/causal_lm/pytorch-0_6b
qwen_3/causal_lm/pytorch-1_7b
qwen_3/causal_lm/pytorch-8b
qwen_3/causal_lm/pytorch-14b
llama/causal_lm/pytorch-llama_3_8b_instruct
llama/causal_lm/pytorch-llama_3_1_8b
```

The remaining 11 models have issues or are incompatible with the current test structure.
1. 
```
mistral/pytorch-mistral_small_24b_instruct_2501
qwen_2_5/causal_lm/pytorch-32b_instruct
qwen_2_5_coder/pytorch-32b_instruct
qwen_3/causal_lm/pytorch-32b
```
Out of memory issues due to https://github.com/tenstorrent/tt-mlir/issues/6524, can be added soon once https://github.com/tenstorrent/tt-mlir/pull/6539 is uplifted into the tt-xla wheel. We also intend to add llama/causal_lm/pytorch-llama_3_3_70b_instruct once OOM issue is resolved.

2. 
```
gemma/pytorch-google/gemma-1.1-7b-it
gemma/pytorch-google/gemma-2-9b-it
falcon/pytorch-tiiuae/falcon-7b-instruct
```
Blocked by https://github.com/tenstorrent/tt-xla/pull/2734.

3. 
```
mistral/pytorch-devstral_small_2505
mistral/pytorch-magistral_small_2506
```
Uses `MistralTokenizer` (instead of `AutoTokenizer`), which does not have the interface expected by the current benchmark test structure.

4. 
```
mistral/pixtral/pytorch
```
Currently does not have a tokenizer

5. 
```
qwen_3/embedding/pytorch-embedding_8b
```
Current test structure expects logits so it is incompatible with embedding models.
